include/ipz_parser.hpp

```diff
@@ -197,6 +215,46 @@ class IpzVpdParser : public ParserInterface
         getRecordDetailsFromVTOC(const types::Record& l_recordName,
                                  const types::RecordOffset& i_vtocOffset);
 
+    /**
+     * @brief API to update record's ECC
+     *
+     * This API is required to update the record's ECC based on the record's
+     * current data.
+     *
+     * @param[in] i_recordDataOffset - Record's data offset
+     * @param[in] i_recordDataLength - Record's data length
+     * @param[in] i_recordECCOffset - Record's ECC offset
+     * @param[in] i_recordECCLength - Record's ECC length
+     * @param[in,out] io_vpdVector - Copy of m_vpdVector to update record's ECC.
+     *
+     * @throw std::runtime_error on ECC failure.
```

Comment:
Code is throwing ECC exception. Need to update here?

Reply:
updated